README.md

```diff
@@ -181,6 +181,8 @@ end
 ```
 
 ### Start Watchdog
+Since we heavily depend on the [listen gem](https://github.com/guard/listen), as a secondary insurance measure we can use a `ProcessSettings::Watchdog` to ensure that a given file path's versions from memory and disk are in sync.
+ 
```

Comment:
@ColinDKelley Does this description sufficiently explain why we need watchdog?

Reply:
Maybe you can merge that with what I proposed? I like being explicit that it's insurance to check if `listen` or the underlying OS drivers has a bug.